x-pack/plugins/security_solution/scripts/check_circular_deps/run_check_circular_deps_cli.js

```diff
@@ -34,6 +34,29 @@ run(
 
     const circularFound = result.circular();
     if (circularFound.length !== 0) {
+      let count = 0;
```

Comment:
- Should I write one image per circular that is found?
- Should I only run this logic if a flag is passed to the script?
- Any other good ideas?

Reply:
I think having it behind a flag makes sense. Depending on the output I can go either way for individual vs grouped images